<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffractionLimitedAnalysis</class>
 <widget class="QMainWindow" name="DiffractionLimitedAnalysis">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Diffraction Limited Analysis</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QGroupBox" name="main_runBox">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>9</y>
        <width>761</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>230</height>
       </size>
      </property>
      <property name="title">
       <string>Current run</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3" colspan="3">
        <widget class="QPlainTextEdit" name="main_pathEntry">
         <property name="plainText">
          <string>Path to folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPlainTextEdit" name="main_sizeEntry">
         <property name="plainText">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="main_runButton">
         <property name="text">
          <string>Run</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="main_pathLabel">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="main_sizeLabel">
         <property name="text">
          <string>Size in pixels</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="main_thresholdLabel">
         <property name="text">
          <string>Threshold in SD</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPlainTextEdit" name="main_thresholdEntry">
         <property name="plainText">
          <string>Threshold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="main_resultBox">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>134</y>
        <width>761</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Current results</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="main_resultTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlotWidget" name="main_particlePlot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="main_oaButton">
         <property name="text">
          <string>Orthogonal analysis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="main_saveresultButton">
         <property name="text">
          <string>Save results</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="orthogonalTab">
     <attribute name="title">
      <string>Orthogonal analysis</string>
     </attribute>
     <widget class="PlotWidget" name="oa_particlePlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="oa_intperareaPlot" native="true">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>401</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="oa_nareaPlot" native="true">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>401</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="oa_controlWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="oa_applyButton">
         <property name="text">
          <string>Apply threshold(s)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="oa_defaultButton">
         <property name="text">
          <string>Reset to default</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="oa_saveresultButton">
         <property name="text">
          <string>Save results</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="LogTextEdit" name="main_logBar">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>470</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mainMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="mainMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">customwidgets.py</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
